packages/webchat/src/components/Header.tsx

```diff
@@ -2,8 +2,9 @@ import { observe } from 'mobx'
 import { inject, observer } from 'mobx-react'
 import React from 'react'
 
-import confirmDialog from '../../../../../../packages/ui-shared-lite/ConfirmDialog'
-import MoreOptions from '../../../../../../packages/ui-shared-lite/MoreOptions'
+// TODO: put this back
+// import confirmDialog from '../../../../../../packages/ui-shared-lite/ConfirmDialog'
+// import MoreOptions from '../../../../../../packages/ui-shared-lite/MoreOptions'
```

Comment:
We can't import this stuff from ui-shared-lite aswell. Do we copy paste this then?